src/dev.sh

```diff
@@ -304,13 +304,21 @@ function package ()
         tar_name="${agent_pkg_name}.tar.gz"
         echo "Creating $tar_name in ${LAYOUT_DIR}"
         tar -czf "${tar_name}" -C ${LAYOUT_DIR} .
+        echo "Calculate SHA256 checksum for $tar_name"
+        if [[ ("$PLATFORM" == "linux") ]]; then
+            sha256sum "${tar_name}" | cut -b 1-64 > ./Checksum_SHA256.txt
```

Comment:
The only thing that concerns me is if we add this file today, and then it becomes inconvenient for us to maintain tomorrow. I hate to pull back on functionality.

User's don't technically *need* this file. They can download to a temp location and then move to the cache location if the download is successful.